CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
```

Comment:
this should be
```suggestion
## [develop]
```
(required for our release process)

Reply:
I was just following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) as the top of the file recommends 😉